src/core/Akka.Persistence/Journal/AsyncWriteJournal.cs

```diff
@@ -76,6 +82,7 @@ protected AsyncWriteJournal()
                     _replayFilterMode = ReplayFilterMode.Warn;
                     break;
                 default:
+                    _hasFatalError = true;
                     throw new ConfigurationException($"Invalid replay-filter.mode [{replayFilterMode}], supported values [off, repair-by-discard-old, fail, warn]");
```

Comment:
Stupid question... if the ctor throws is there even a chance this variable will be read?

Reply:
That's a Jon Skeet question - I honestly have no idea.

Reply:
As it turns out: https://www.youtube.com/watch?v=BxgaXOSS2t4

So I got rid of this.